app/pages/lab-pages-editor/components/WorkflowHeader.jsx

```diff
@@ -27,18 +52,67 @@ export default function WorkflowHeader({
         <ReturnIcon />
         {strings.PagesEditor.components.WorkflowHeader.return}
       </Link>
-      <div className="flex-row flex-item justify-around">
-        <button className="unselected" type="button" onClick={onClick}>
-          {strings.PagesEditor.components.WorkflowHeader.tasks}
-        </button>
-        <button className="selected" type="button" onClick={onClick}>
-          {strings.PagesEditor.components.WorkflowHeader.workflow_settings}
-        </button>
+      <div
+        role="tablist"
+        className="flex-row flex-item justify-around"
+      >
+        {tabs.map((tab, index) => (
+          <TabButton
+            id={tab.id}
+            index={index}
+            key={`${tab.id}`}
+            label={tab.label}
+            onClick={onClick}
+            onKeyUp={onKeyUp}
+            selected={(currentTab === index)}
+          />
+        ))}
       </div>
     </div>
   );
 }
 
 WorkflowHeader.propTypes = {
-  projectId: PropTypes.string
+  currentTab: PropTypes.number,
+  projectId: PropTypes.string,
+  setCurrentTab: PropTypes.func,
+  tabs: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      label: PropTypes.string
+    })
+  )
+};
+
+function TabButton({
+  id,
+  index,
+  label = '',
+  onClick = () => {},
+  onKeyUp = () => {},
+  selected = false
+}) {
+  return (
+    <button
+      aria-selected={selected}
```

Comment:
You can add `aria-controls=TAB-PANEL-ID` to link each tab to its corresponding tab panel.